### PR TITLE
Translations Cleanup

### PR DESCRIPTION
**Pull request by Abby-Wheelis**
- Add missing error keys
- correct mistakes in the Spanish translations
- ~~remove stale multilabel keys~~ Reverted based on the discussion [in this thread](https://github.com/e-mission/e-mission-translate/pull/47#discussion_r1367569887)

### CODE REVIEW

File: lo/i18n/lo.json [hunk 1]
```diff
@@ -398,9 +398,15 @@
     "errors": {
         "registration-check-token": "ການລົງທະບຽນຜູ້ໃຊ້ຜິດພາດ. ກະລຸນາກວດເບິ່ງໂທເຄັນຂອງທ່ານແລະລອງໃໝ່ອີກ.",
         "not-registered-cant-contact": "ຜູ້ໃຊ້ບໍ່ໄດ້ລົງທະບຽນ, ດັ່ງນັ້ນເຄື່ອງແມ່ຂ່າຍບໍ່ສາມາດຕິດຕໍ່ໄດ້.",
+        "while-initializing-label": "While initializing Label tab: ",
+        "while-loading-pipeline-range": "Error while loading pipeline range",
```
Comment (shankari) [inline lo/i18n/lo.json:402]: Can we fill these in with translations from Google Translate on the principle that something is better than nothing?

File: es/i18n/es.json [hunk 2]
```diff
@@ -397,9 +397,15 @@
     "errors": {
         "registration-check-token": "Error de registro de usuario. Por favor verifique su token e inténtelo nuevamente.",
         "not-registered-cant-contact": "El usuario no está registrado, por lo que no se puede contactar con el servidor.",
+        "while-initializing-label": "While initializing Label tab: ",
+        "while-loading-pipeline-range": "Error while loading pipeline range",
```
Comment (shankari) [inline es/i18n/es.json:401]: These have to be the spanish translations, right?

**Review by shankari — CHANGES_REQUESTED** [review]

File: lo/i18n/lo.json (hunk 1 above)
Comment (Abby-Wheelis) [reply lo/i18n/lo.json]: I just made these updates in [ef9628e](https://github.com/e-mission/e-mission-translate/pull/52/commits/ef9628e23bacd9dbdf37a12c0867aa187a62fd86)

File: es/i18n/es.json (hunk 2 above)
Comment (Abby-Wheelis) [reply es/i18n/es.json]: updates made in [ef9628e](https://github.com/e-mission/e-mission-translate/pull/52/commits/ef9628e23bacd9dbdf37a12c0867aa187a62fd86)

**Review by shankari — APPROVED** [review]

**Pull request merged**